### PR TITLE
refactor(ui): reuse ConfirmScreen in HideTokenScreen

### PR DESCRIPTION
**Pull request by jasonzhouu**
Implements https://github.com/argentlabs/argent-x/issues/293

https://user-images.githubusercontent.com/19926174/155840471-eb232766-6c8c-4ac8-a3c3-2aab9ed02d32.mov

### CODE REVIEW

File: packages/extension/src/ui/screens/HideTokenScreen.tsx [hunk 1]
```diff
@@ -35,43 +26,33 @@ export const HideTokenScreen: FC = () => {
 
   const { name } = toTokenView(token)
 
-  return (
-    <>
-      <Header>
-        <BackButton />
-      </Header>
+  const handleSubmit = async () => {
```
Comment (delaaxe) [inline packages/extension/src/ui/screens/HideTokenScreen.tsx:29]: Does this function need to be async?

File: packages/extension/src/ui/screens/HideTokenScreen.tsx [hunk 2]
```diff
@@ -35,43 +26,33 @@ export const HideTokenScreen: FC = () => {
...
+    try {
+      removeToken(token)
+      navigate(routes.account())
+    } catch (e) {
```
Comment (delaaxe) [inline packages/extension/src/ui/screens/HideTokenScreen.tsx:33]: Hide error variable if unused
```suggestion
    } catch {
```

**Review by delaaxe — CHANGES_REQUESTED** [review]
This is much cleaner, thank you

File: packages/extension/src/ui/screens/HideTokenScreen.tsx (hunk 1 above)
Comment (jasonzhouu) [reply packages/extension/src/ui/screens/HideTokenScreen.tsx]: Thank you for your review. 
Updated

**Review by delaaxe — APPROVED** [review]

**Pull request merged**